packages/dev/sharedUiComponents/src/lines/optionsLineComponent.tsx

```diff
@@ -46,6 +47,11 @@ export class OptionsLineComponent extends React.Component<IOptionsLineComponentP
     }
 
     constructor(props: IOptionsLineComponentProps) {
+        // Initialize default props
+        props = {
+            ...props,
+            fallbackValue: -1,

```

Comment:
ahhhhhh I merged too early :-) this willl always put fallbackValue to -1 ??? fallbackvalue should be before isnt it ?